src/libraries/System.IO.Packaging/src/System/IO/Packaging/OrderedDictionary.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.IO.Packaging
+{
+    /// <summary>
+    /// A collection that ensures uniqueness among a list of elements while maintaining the order in which the elements were added.
+    /// This is similar to <see cref="OrderedDictionary{TKey, TValue}"/>, but the items will not be sorted by a comparer but rather retain the
+    /// order in which they were added while still retaining good lookup, insertion, and removal.
+    /// </summary>
+    internal class OrderedDictionary<TKey, TValue> : IEnumerable<TValue>
+    {
+        private readonly Dictionary<TKey, LinkedListNode<TValue>> _dictionary;
+        private readonly LinkedList<TValue> _order;
+
+        public OrderedDictionary(int initialCapacity, IEqualityComparer<TKey> comparer)
```

Comment:
Is the comparer actually needed?  The only place that this appears to be instantiated has StringComparer.Ordinal passed in, which is the same behavior you'd get if you didn't specify a comparer.

Reply:
Cool - I didn't know that by default it was an ordinal comparer. I assumed it was culture dependent like a lot of the default string operations. I'll remove it.